src/model/proto/checklist.proto

```diff
@@ -26,11 +26,17 @@ message ChecklistGroup {
   string title = 1;
   repeated Checklist checklists = 2;
 
+  /*
+  These categories are currently used for output in the ForeFlight format. The values are in lower case because that is
+  what ForeFlight expects. Also, only these three categories are expected to be present. If you extend or otherwise
+  change this enum, be sure to create an appropriate mapping for the ForeFlight format.
+
+  TODO: Make keys uppercase when ts-protobuf supports customizing JSON output to match ForeFlight's expectations.
+  */
   enum Category {
-    UNKNOWN = 0;
-    NORMAL = 1;
-    ABNORMAL = 2;
-    EMERGENCY = 3;
+    normal = 0;
```

Comment:
Totally optional, but FYI now that I saw your commit changing the JSON output for this: the reason for the UNKNOWN or UNSPECIFIED value is so that if you forget it, it doesn't look the same as if it's set to the first value (since in proto3 there's no required vs optional fields like in proto2).
https://protobuf.dev/programming-guides/dos-donts/#unspecified-enum
(internally we actually prevent proto changes that don't follow this, but I think in this case it's pretty harmless if it's set to "normal" accidentally)

Reply:
This is indeed how I started for the reasons you mention, but then decided to back out and go against the rules to make it easier and safer(?). I implemented a conforming version in df2a24d. This has the consequence that all loading should be routed through the format class (which I guess is a good thing!), so I can explicitly make `normal` the default.

Otherwise I can add proper support for `unknown`, including a selection in the UI, but this will inevitably lead to questions like "why is my exported ForeFlight checklist empty? Or go back to breaking the rules.

I'm not sure what's best, I can live with both options, but I would be unhappy to implement `unknown` as a true default.

Reply:
I think what you did is right - unknown should be a synonym for "programming error" in this case, not something the user should ever have to deal with.